CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## [Unreleased]
+
+### Added
+
+- Support to get thumbnail images via `Player.getThumbnail` and `Source.getThumbnail`
```

Comment:
I would prefer the changelog style that you used in #224 

```suggestion
- `Player.getThumbnail` and `Source.getThumbnail` APIs to get thumbnail images
```

Reply:
Good point!
Taking the suggested change!